### ISSUE TITLE
streamdeck-ui fails to run on NixOS Unstable - Could not find the Qt platform "xcb"

### ISSUE THREAD

**Issue by RodneyBrown** [NixOS/nixpkgs#232105]
### Describe the bug
When I switch NixOS from 22.11 to Unstable:
`nixos-rebuild switch --flake .`
Builds without issue - however when I run streamdeck-ui, it produces the following error:
`qt.qpa.plugin: Could not find the Qt platform plugin "xcb" in ""`

### Steps To Reproduce
1. Switch NixOS from 22.11 to Unstable branch
2. Try to run streamdeck-ui

### Expected behavior
streamdeck-ui GUI should appear and the Streamdeck itself should light up with programmed buttons.

### Screenshots
N/A - Error message described above.

### Additional context
If I switch back to 22.11 and rebuild, streamdeck-ui works again.

### Notify maintainers
@SebTM 
@Majiir 

### Metadata
Please run `nix-shell -p nix-info --run "nix-info -m"` and paste the result.

```console
$ nix-shell -p nix-info --run "nix-info -m"
 - system: `"x86_64-linux"`
 - host os: `Linux 6.3.1, NixOS, 23.05 (Stoat), 23.05.20230514.0470f36`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.13.3`
 - channels(root): `"home-manager-22.11.tar.gz, nixos-22.11"`
 - nixpkgs: `/nix/var/nix/profiles/per-user/root/channels/nixos`
```

**Comment by SebTM** [comment]
Should be resolved by #226525 but is waiting on review :)

**Comment by SebTM** [comment]
Resolved with mentioned PR got merged
